GitCommands/Gpg/GitGpgController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using GitCommands.Config;
+using GitCommands.Gpg;
+using GitExtUtils;
+
+namespace GitCommands.Git.Gpg
+{
+    partial class GitGpgController : IGitGpgController
```

Comment:
The class is in the Git namespace and is the driver between Git and Gpg functionality.  

![image](https://github.com/gitextensions/gitextensions/assets/3629489/da160f68-9226-474e-bca0-bff672913b82)

I split so that the implimentation of the gpg key functionality is defined with  the gpg key providing functionality.  I was thinking of splitting into own class but am 2 ways on it.  Outside the user sees a cohesive this thing drives Git/Gpg interaction and the developers can navigate to the right pieces easily.   
![image](https://github.com/gitextensions/gitextensions/assets/3629489/e660bee9-9049-401d-bb1f-56e99e8cd23f)